### PR TITLE
Updated example.py

### PR DESCRIPTION
**Pull request by nouni19**
Hello, I updated example.py (and README, only where it shows the example). I added a homework example, and additional comments on the client login successful check. I had to commit twice because I accidentally removed a comment, so sorry for the confusion

### CODE REVIEW

File: example.py [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
     periods = client.periods
 
     for period in periods:
         for grade in period.grades:  # iterate over all the grades
-            print(f'{grade.grade}/{grade.out_of}')  # print out the grade in this style: 20/20
\ No newline at end of file
+            print(f'{grade.grade}/{grade.out_of}')  # print out the grade in this style: 20/20
+
+    today = datetime.date.today() #store today's date using datetime built-in library
+    homework = client.homework(today) #get list of all homework which are for today and after
```
Comment (bain3) [inline example.py:23]: I'd change the comment to: get list of homework for today or later

File: example.py [hunk 2]
```diff
@@ -1,20 +1,32 @@
...
+            print(f'{grade.grade}/{grade.out_of}')  # print out the grade in this style: 20/20
+
+    today = datetime.date.today() #store today's date using datetime built-in library
+    homework = client.homework(today) #get list of all homework which are for today and after 
+    
+    for hw in homework: #iterate through the list
+        print(f"({hw.subject.name}): {hw.description}") #print the homework's subject, title and description
+
+    
+else: #if it didn't log in
```
Comment (bain3) [inline example.py:29]: I don't think we need a comment here, it's pretty straight forward

**Review by bain3 — CHANGES_REQUESTED** [review]
Nice! I would keep this in `example.py` but I don't think we need it in the README. It's a bit long. Also please add spaces in the comments after the `#`. Thanks!

**Comment by nouni19** [comment]
I updated the PR: reverted README changes, removing the "else:" comment and adding spaces after #s

**Pull request merged**